README.md

```diff
@@ -1,7 +1,17 @@
-# starter-kit
+# WICG Starter kit
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/WICG/starter-kit.svg)](https://greenkeeper.io/)
-A simple starter kit for [WICG](https://wicg.io) specs.
+Welcome!  This is a simple starter kit for writing [WICG](https://wicg.io) specifications, and adding new awesome features to the web platform.  In this repo you'll find templates and guides for using them.
+
+## Process
+
```

Comment:
I think step 1 is opening up discussion on Discourse. Getting momentum there with framing the use cases, etc. They can include an explainer, but it's not a requirement:

For example:
https://discourse.wicg.io/t/rfc-proposal-for-face-detection-api/1642